src/utils/helpers/browser/browser.js

```diff
@@ -89,6 +98,16 @@ const Browser = {
   },
 
   /**
+   * Sets the body scroll
+   *
+   * @param {Boolean} enabled
+   * @method setBodyScroll
+   */
+  setBodyScroll: (enabled) => {
+    Browser.getBody().style.overflow = enabled ? '' : 'hidden';
```

Comment:
@SeanArmstrong should this live in a `browser.css` file ?
It sounds weird to add css within the `helper` folder